addons/binding/org.openhab.binding.tplinksmarthome.test/src/test/java/org/openhab/binding/tplinksmarthome/handler/SmartHomeHandlerTest.java

```diff
@@ -99,16 +99,31 @@ public void testInitializeShouldCallTheCallback() throws InterruptedException {
     }
 
     @Test
-    public void testHhandleCommandRefreshType() {
+    public void testHandleCommandRefreshType() {
+        handler.initialize();
+        assertHandleCommandRefreshType(-53);
```

Comment:
Should this also be `expectedRssi`

Reply:
I'm not sure what you mean with this comment about `expectedRssi`?